open_academy/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Open Academy",
+
+    'summary': "Exercise of creation of a module",
+
+    'author': "Emilio Serna, Vauxoo",
+    'website': "https://github.com/EmilioSerna/module-creation",
+
+    'category': 'Uncategorized',
+    'version': '15.0',
+
+    'depends': ['base'],
+
+    'data': [
+        'views/views.xml',
+        'views/templates.xml',
+    ],
+
+    'demo': [
+        'demo/demo.xml',
+    ],
+}
```

Comment:
Use pre-commit to give the correct format to this part... Keep consistent if you will use single quote do it everywhere, same for double quotes, a good practice and pre-commit implementation it's use double quotes.